src/main/resources/db/migration/all/20240118152118__add_missing_application_indexes.sql

```diff
@@ -0,0 +1,12 @@
+CREATE INDEX ON application_timeline_notes(application_id);
+
+CREATE INDEX ON applications(created_by_user_id);
+CREATE INDEX ON applications(schema_version);
```

Comment:
i'm not sure this one is required as I dont _think_ we'll ever be searching for applications with a particular schema (or traversing a hibernate relationship from JsonSchemaEntity to Applications?

Reply:
Ah, good point, I'll get rid 👍